backend/btrixcloud/models.py

```diff
@@ -1093,11 +1093,11 @@ def get_crawl_file(self, storage: StorageRef):
     def prepare_filename(self, filename):
         """prepare filename by sanitizing and adding extra string
         to avoid duplicates"""
-        name = sanitize_filename(filename.rsplit("/", 1)[-1])
-        parts = name.split(".")
+        name, ext = os.path.splitext(filename)
+        name = slugify(filename.rsplit("/", 1)[-1])
```

Comment:
```suggestion
        name = filename.rsplit("/", 1)[-1]
        name = slugify(name.rsplit(".", 1)[0])
```

Avoids the wacz filename being included in both the slugified name and the extension.

e.g. filename "my upload.wacz"

Before change: "upload-{archived item id}-my-upload-wacz-{randstr}.wacz"
After "upload-{archived item id}-my-upload-{randstr}.wacz"

I think we could probably drop the randstr at the end as well, given that we append the upload id to the prefix at the beginning of the filename and so we wouldn't have name conflicts anyway.

Reply:
Thanks for catching that, updated to use name instead of filename in slugify. Removed random string.